python/GafferUI/CompoundEditor.py

```diff
@@ -108,6 +144,40 @@ def nodeSetMenuSignal( cls ) :
 
 		return CompoundEditor.__nodeSetMenuSignal
 
+	def _detachedPanels( self ) :
```

Comment:
Why this accessor? As far as I can tell, nothing else needs protected access to the panels, so we're just adding an unnecessary level of indirection (and then half the time we're ignoring the accessor method and directly accessing the member anyway).

Reply:
Test...

Reply:
Ah, OK. If we can't contrive something so the tests don't need access, then let's add a comment saying "this is for the tests, not for public consumption"...

Reply:
Annotated in https://github.com/GafferHQ/gaffer/pull/3167/commits/4361ca224e657be1db373d6d0fe91d18a252c235